bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/binding/firmware/ProgressCallback.java

```diff
@@ -74,4 +75,14 @@
      * Callback operation to indicate that the firmware update was canceled.
      */
     void canceled();
+    
+    /**
+     * Callback operation to indicate that the percentage progress of the firmware update has changed. 
```

Comment:
> So there are several mechanism now to inform about an update progress.... either using next operation or setPercentageProgress...

There is no coherence to the defindeSequence operation. Percentage progress is not bound to ProgressSteps. Maybe there is a coherence but this is implementation specific.